R/geom-.r

```diff
@@ -147,6 +147,17 @@ Geom <- gganimintproto("Geom",
   pre_process = function(g, g.data, ranges){
     list(g = g, g.data = g.data)
   },
+  
+  get_off_params = function(input_string) {
+    pattern <- "_off$"
+    # Return parameter name without the _off suffix
+    if (grepl(pattern, input_string)) {
+      result <- sub(pattern, "", input_string)
+      return(result)
+    } else {
+      return("")
```

Comment:
it is simpler to not use return
also this whole function could be replaced by grep(input_string, pattern, value=TRUE)

Reply:
Thanks for the suggestion. I rewrote the check using grep and sub and the whole function is not required anymore